### ISSUE TITLE
1.3.5.3 Server Pack Missing some mods

### ISSUE THREAD

**Issue by Honobe** [adam9899/Issues#484]
When I updated and downloaded the new server pack I was missing my dank null. When I looked in the mods folder it was missing dank null.

**Comment by NordicGamerFE** [comment]
Its not missing its been downgraded to a other version please read the changelog next time

**Comment by The-Brandolorian** [comment]
Incorrect, the Dank Null file is missing from the server pack download. Probably should have checked before shutting him down haha.
I used the one from my SP folder

**Comment by Avalonzo** [comment]
Duplicate of #479

**Comment by Avalonzo** [comment]
Please download 1.5.3 server files again as dank/null/ was added back in to the files.

**Comment by Avalonzo** [comment]
Someone just noticed the link on curseforge was still for the wrong server files here is the correct link - https://www.curseforge.com/minecraft/modpacks/minecraft-eternal/files/2932890.
